data/projects/p0tion.ts

```diff
@@ -9,15 +9,18 @@ const content: ProjectContent = {
 
 export const p0tion: ProjectInterface = {
   id: "p0tion",
-  section: "pse",
-  projectStatus: ProjectStatus.ACTIVE,
+  section: "archived",
+  projectStatus: ProjectStatus.INACTIVE,
   content,
   image: "p0tion.png",
   name: "p0tion",
   links: {
     website: "https://ceremony.pse.dev/",
     github: "https://github.com/privacy-scaling-explorations/p0tion",
   },
+  // extraLinks: {
+  //   learn: [{ label: "Trusted Setup/P0tion LTS Announcement", url: "TODO" }],
+  // },
```

Comment:
@NicoSerranoP is preparing something we will publish to communicate this.
So to be update later